boards/ESP8266_BOARD.py

```diff
@@ -28,7 +28,7 @@
   'flash'   : 512,
   'speed'   : 80,
   'usart'   : 1,
-  'spi'     : 0,
+  'spi'     : 1,
```

Comment:
I agree ... there are a couple of issues floating around just now ...  see Issue #675 for the one that is most relevant.